user-interface/src/main/java/life/qbic/datamanager/views/projects/project/measurements/MeasurementMain.java

```diff
@@ -646,9 +717,26 @@ private static List<MeasurementRegistrationInformationNGS> mergeByPool(
     return finalMeasurements;
   }
 
-  private void displayRegistrationSuccessful() {
+
+  private void processRegistrationResults(int numberOfSuccesses, int numberOfRequests) {
```

Comment:
This method assumes that it is called after all computation is performed or failed.
Maybe you can add a one line comment to indicate that assumption?